ncm-iptables/src/main/perl/iptables.pm

```diff
@@ -852,16 +852,9 @@ sub Configure($$@) {
     ######################################################################
     # Create tmpdir if necessary
     ######################################################################
-    my $tmpdir = "/var/tmp/ncm/";
-    mkpath($tmpdir, 0, 0755 ) unless ( -e $tmpdir );
+    my ($iptc_temp_fh, $iptc_temp) = tempfile();
```

Comment:
has to use `eval{}` to catch any failures (or the following line makes no sense).